spec/requests/providers/other_assets_controller_spec.rb

```diff
@@ -204,7 +204,22 @@
                 expect(application.savings_amount?).to be false
                 expect(response).to redirect_to(providers_legal_aid_application_policy_disregards_path(application))
               end
+
+              context 'when the calculation date is prior to the policy disregards date' do
+                let(:application) do
+                  create :legal_aid_application,
+                         :with_positive_benefit_check_result,
+                         used_delegated_functions: true,
+                         used_delegated_functions_on: Date.new(2021, 1, 5)
+                end
+                let(:policy_disregards) { false }
+
+                it 'redirects to policy disregards' do
```

Comment:
Should this be 'redirects to check your answers'?

Reply:
I don't think so, checking answers is 165-188
This is when they select nothing on their first runthrough and have a delegated function date before the cut-over
(mocked with `receive(:capture_policy_disregards?).and_return(false)`

Reply:
Ah, OK. I just noticed that the test checks for redirection to redirect_to(providers_legal_aid_application_check_passported_answers_path(application)), so it seemed like there was a bit of a mismatch between the description and the assertion.